cluster/ci/config/prow/jobs/README.md

```diff
@@ -0,0 +1,130 @@
+ci-operator templates
+=====================
+
+
+E2E tests
+---------
```

Comment:
I was thinking about the doc structure and purpose, and I think the purpose should be to *describe how to set up E2E tests using ci-operator's template feature*. Therefore, I think the H1 should be about E2E tests, followed by a short paragraph about what is the doc about, then dive into templates.

Also, maybe elaborate E2E at least once near the begging. I assume it means end-to-end tests, but I am not sure every newcomer knows it.

Reply:
Yes, I forgot to mention I wasn't very happy with the directory/title
structure.  It is much better now that the templates are under the
`ci-operator` directory.

Updated the heading.